api_app/resources/strings.py

```diff
@@ -54,8 +54,10 @@
 API_GET_USER_RESOURCE_TEMPLATES = "Get user resource templates applicable to the workspace service template"
 API_GET_USER_RESOURCE_TEMPLATE_BY_NAME = "Get user resource template by name and workspace service"
 
+# cost report
 API_GET_COSTS = "Get overall costs"
 API_GET_WORKSPACE_COSTS = "Get workspace costs"
+API_GET_COSTS_INVALID_TIME_PERIOD = "The time period for pulling the data cannot exceed 1 year(s)"
```

Comment:
If you cant pull more than 1 year - do you need the plural on the end `(s)`

Reply:
I don't :) actually copied the string from azure cost management REST API, didn't notice that....
I'll remove the plural, thanks.